modules/nw-about-static-egress-ips.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * networking/enabling-static-egress-ips.adoc
+
+[id="about-{context}"]
+= About static IPs for external project traffic
+
+As a cluster administrator you can assign specific, static IP
+addresses to projects, so that traffic is externally easily recognizable. This
+is different from the default egress router, which is used to send traffic to
+specific destinations.
+
+Recognizable IP traffic increases cluster security by ensuring the origin is
+visible. Once enabled, all outgoing external connections from the specified
+project will share the same, fixed source IP, meaning that any external
+resources can recognize the traffic.
+
+Unlike the egress router, this is subject to `EgressNetworkPolicy` firewall
+rules.
```

Comment:
It might make sense to rearrange things so this ends up with the other egress-router-related comment.

Actually, maybe there should just be an introduction to egress solutions section, explaining the egress router and the two styles of egress IPs, and then more detailed subsections for each of them.